app/components/signinModal/tests/index.test.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Modal } from 'react-bootstrap';
+import ReactTestUtils from 'react/lib/ReactTestUtils';
+import SigninModal from 'components/signinModal';
+
+describe('Signin Modal', () => {
+  const state = {
+    isModalOpen: true,
+    user: {
+      email: '',
+      password: ''
+    }
+  };
+  const signinModalComponent = mount(<SigninModal />);
+  signinModalComponent.setState(state);
+  const modalDialogContent = signinModalComponent.find(Modal).node._modal.getDialogElement();
+
+  it('renders Modal with form fields', () => {
+    expect(signinModalComponent.find(Modal).length).toEqual(1);
+    expect(modalDialogContent.getElementsByClassName('form-group').length).toEqual(2);
+    expect(modalDialogContent.getElementsByClassName('form-control').length).toEqual(2);
+  });
+
```

Comment:
add `context` to clearify specs semantic. Like:

context(‘when email is changed’, () => {
    it(‘it includes error class’, () => {